weave/trace_server/remote_http_trace_server.py

```diff
@@ -67,23 +67,29 @@ def _is_retryable_exception(e: Exception) -> bool:
 
 
 def _log_retry(retry_state: tenacity.RetryCallState) -> None:
-    logger.info(
-        "retry_attempt",
+    logger.warning(
+        "Retry attempt %d for function '%s'",
+        retry_state.attempt_number,
+        retry_state.fn.__name__,
         extra={
-            "fn": retry_state.fn,
-            "attempt_number": retry_state.attempt_number,
             "exception": str(retry_state.outcome.exception()),
```

Comment:
wouldn't the exception be the most useful?

Reply:
i feel like showing the exception every retry could blow up console
And since we see exception on failure, i thought no need to show on every retry

Reply:
after reading the code a bit more I am finding that we rery every 5 mins for up to 36 hours, it makes sense to show exception